### DISCUSSION TITLE
rspamd set server_name and config for servers

### DISCUSSION THREAD

**Discussion by Klar** [docker-mailserver/docker-mailserver#4100] [Q&A] [answered]
Hi,

I want to set my correct `server_name` and configure the cluster for the webinterface for both of my servers.

From my understanding I can use `config/rspamd/override.d/worker-controller.inc` but this will overwrite the current worker-controller file . This is fine I think, I just have to track if there is some new configuration in there which I need to add as well.

For `server_name` I would need to edit `options.inc` as I don't think I should fully overwrite the file. There is also the option to use the folder `local.d` which is more of a configuration merge.

How do I use the `local.d` with my configuration in files `worker-controller.inc` and config for `local.d`?

I tried to mount (docker volume) the `local.d` folder into the container `/etc/rspam.d/local.d` but I got an error about missing file `greylist.conf`. Is this something docker-mailserver already doing and I should not change that?

**Comment by polarathene** [comment]
@georglauterbach just pinging you here in case the docs don't explain this well, since you're wanting to switch rspamd to default in the next major release.

@Klar I don't deal with the rspamd support, so I'm not able to assist much. Are you still using DMS? Did you manage to resolve this on your own?

**Comment by georglauterbach** [comment, marked as answer]
This is all very well documented [here](https://docker-mailserver.github.io/docker-mailserver/edge/config/security/rspamd/#with-the-help-of-a-custom-file). You probably do not need to write a single file yourself.

**Reply by georglauterbach** [reply]
Moreover, DMS does not use `override.d/`, but only `local.d`, to allow complete overwrites by the user; so in case you really need it, you can always use it.
